packages/conf-rust-1-53/conf-rust-1-53.0.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "david@o1labs.org"
+authors: "David Wong"
+homepage: "https://github.com/mimoo/conf-rust-1-53"
+bug-reports: "https://github.com/mimoo/conf-rust-1-53/issues"
+dev-repo: "git+https://github.com/mimoo/conf-rust-1-53.git"
+license: "MIT"
+build: [
+  ["cargo" "--version"]
+  ["cargo" "+1.53.0" "build"]
```

Comment:
This package can basically be summerised by this command. If it’s a requirement of your package, why not add this check to your configure/build step early and add a meaningful error there? If opam-repository had a package for all the possible preconditions (bash version, gcc version, autoconf version, …) it wouldn’t be manageable in the current state.